src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -102,6 +102,8 @@ public class ServerStore : IDisposable
         internal readonly SizeLimitedConcurrentDictionary<string, ConcurrentQueue<DateTime>> ClientCreationRate = 
             new SizeLimitedConcurrentDictionary<string, ConcurrentQueue<DateTime>>(50);
 
+        internal bool IgnoreProcessorAffinityChanges = false;
```

Comment:
this will affect whole process anyway, I do not see a point here

Reply:
done